webapp/components/_new/generic/LabeledButton/LabeledButton.story.js

```diff
@@ -0,0 +1,22 @@
+import { storiesOf } from '@storybook/vue'
+import helpers from '~/storybook/helpers'
+import LabeledButton from './LabeledButton.vue'
+
+helpers.init()
+
+storiesOf('Generic/LabeledButton', module)
+  .addDecorator(helpers.layout)
+  .add('default', () => ({
+    components: { LabeledButton },
+    data: () => ({
+      filled: false,
+    }),
+    template: `
+      <labeled-button
+        icon="check"
+        :filled="filled"
+        label="Toggle Me!!"
+        @click="filled = !filled"
+      />
+    `,
+  }))
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-03-24T17:06:02Z" title="Tuesday, March 24th 2020, 6:06:02 pm +01:00">Mar 24, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,35 @@
+import { storiesOf } from '@storybook/vue'
+import helpers from '~/storybook/helpers'
+import LabeledButton from './LabeledButton.vue'
+
+storiesOf('Generic/LabeledButton', module)
+  .addDecorator(helpers.layout)
+  .add('default', () => ({
+    components: { LabeledButton },
+    data: () => ({
+      filled: false,
+    }),
+    template: `
+      <labeled-button
+        icon="check"
+        :filled="filled"
+        label="All"
+        @click="filled = !filled"
+      />
+    `,
+  }))
+
+  .add('filled', () => ({
+    components: { LabeledButton },
+    data: () => ({
+      filled: true,
+    }),
+    template: `
+      <labeled-button
+        icon="check"
+        :filled="filled"
+        label="All"
+        @click="filled = !filled"
+      />
+    `,
+  }))
```


@mattwr18 cool that you added the story! 💚 

I think, since it's interactive, we don't really need the `filled` story and could instead maybe give the button in the `default` story the label `Toggle me` to make it clear that the button is reacting to input? What do you think?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-24T19:55:37Z" title="Tuesday, March 24th 2020, 8:55:37 pm +01:00">Mar 24, 2020</time>_

---


Great idea!